wfe-core/src/main/java/ru/runa/wfe/chat/decoder/MessageRequestBinaryDecoder.java

```diff
@@ -9,23 +9,23 @@
 import javax.websocket.EndpointConfig;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.ejb.interceptor.SpringBeanAutowiringInterceptor;
-import ru.runa.wfe.chat.dto.ChatDto;
+import ru.runa.wfe.chat.dto.request.MessageRequest;
 
 @Interceptors({SpringBeanAutowiringInterceptor.class})
-public class ChatDtoBinaryDecoder implements Decoder.Binary<ChatDto> {
+public class MessageRequestBinaryDecoder implements Decoder.Binary<MessageRequest> {
 
     @Autowired
     private ObjectMapper objectMapper;
 
     @Override
-    public ChatDto decode(ByteBuffer byteBuffer) throws DecodeException {
-        ChatDto chatDto;
+    public MessageRequest decode(ByteBuffer byteBuffer) throws DecodeException {
+        MessageRequest dto;
         try {
-            chatDto = objectMapper.readValue(byteBuffer.array(), ChatDto.class);
+            dto = objectMapper.readValue(byteBuffer.array(), MessageRequest.class);
```

Comment:
```suggestion
            return objectMapper.readValue(byteBuffer.array(), MessageRequest.class);
```
? и нижний return убрать просто